wger/exercises/templates/exercise/view.html

```diff
@@ -253,6 +253,18 @@ <h5 class="mt-4">{% trans "Weight log" %}:</h5>
     {% render_weight_log logs svg_uuid user %}
 {% endif %}
 
+<h5 class="mt-4">{% trans "Exercise Variations" %}:</h5>
+{% if exercise.variations %}
+    <ul>
+    {% for variation in exercise.variations.all %}
+        <li>
+            <a href="{{ variation.get_absolute_url }}">{{ variation }}</a>
+        </li>
+    {% endfor %}
+    </ul>
+{% else %}
+    <p>{% trans "None" %}</p>
+{% endif %}
 {% endblock %}
```

Comment:
I'd wrap the whole section in the {% if %} and not show anything if there aren't any variations